src/app/modules/webhook/webhook.service.ts

```diff
@@ -243,21 +251,29 @@ export const createInitialWebhookSender =
   > => {
     // Attempt to send webhook
     return sendWebhook(submission.getWebhookView(), webhookUrl).andThen(
-      (webhookResponse) =>
+      (webhookResponse) => {
+        webhookStatsdClient.increment('sent', 1, 1, {
+          responseCode: webhookResponse.response.status.toString(),
+          webhookType: getWebhookType(webhookUrl),
+        })
```

Comment:
Additional note for tags:
* We should probably report a boolean tag for `is_retry: 0|1` so we can filter out retries
* Any way we can somehow know the agency?

Reply:
This DD tracking is in the `createInitialWebhookSender`, so it only counts the initial webhook that's sent, which means we don't have to filter out retries. I have added another tag `isRetryEnabled` in case we want to know whether retries have been enabled on the different categories of webhooks.

Including agency - we only have the submission ID in this webhook service, so I'd have to add a query upstream in `performEncryptPostSubmissionActions` to check for the admin and agency. I've left it for now as the agency info wasn't requested by Amit, but can definitely add it in if it's information that we'd like in the future!